spec/requests/admin/registries_controller_spec.rb

```diff
@@ -41,7 +27,7 @@
           when there's something wrong with the reachability of the registry" do
         allow_any_instance_of(Registry).to receive(:reachable?).and_return("Error")
         expect do
-          post :create, registry: attributes_for(:registry)
+          post "/admin/registries", params: { registry: attributes_for(:registry) }
```

Comment:
Doesn't `admin_registries_url` work in this case?

Reply:
For whatever reason, this `_url` helper only worked for me on requests for `get` requests, and nothing worked for post/put/etc. methods. I'm not sure why, to be honest, but I can give it another look if you want :wink:

Reply:
I've seen that being used on `put` somewhere else, almost sure...